packages/cli/src/commands/deploy-v2.ts

```diff
@@ -2,12 +2,12 @@ import chalk from "chalk";
 import glob from "glob";
 import path, { basename } from "path";
 import type { CommandModule, Options } from "yargs";
-import { loadWorldConfig } from "../config/world/index.js";
+import { loadStoreConfig, loadWorldConfig } from "@latticexyz/config";
+import { MUDError } from "@latticexyz/config";
```

Comment:
initially wanted to put `MUDError` into `utils` package, but importing it doesn't work yet so left this for another PR